source/extensions/filters/http/ext_proc/client_impl.cc

```diff
@@ -0,0 +1,64 @@
+#include "extensions/filters/http/ext_proc/client_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+static constexpr char kExternalMethod[] =
+    "envoy.service.ext_proc.v3alpha.ExternalProcessor.Process";
+
+ExternalProcessorClientImpl::ExternalProcessorClientImpl(
```

Comment:
FYI, we're going to want to have a thread-local cache of aync clients for ext_proc similar to what is done for the gRPC access loggers and ext_authz. This is what I referenced in my comment on your doc a while back. But, I'm hoping that we can land https://github.com/envoyproxy/envoy/issues/12598 soon to avoid you needing to have to do this explicitly.

Reply:
I see that in the ext_authz filter and I was planning to first get the basic thing working and then add that. However I'm watching that PR and if it comes in soon, then even better.